.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,33 @@
+name: Feature request
+description: Propose a change to Yaegi
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ⚠️ If you do not respect this template your issue will be closed.
```

Comment:
Idem as above, please remove this warning.

Reply:
This warning was in the previous template and added by you in #333